src/main/java/com/bunq/sdk/exception/ApiException.java

```diff
@@ -1,37 +1,30 @@
 package com.bunq.sdk.exception;
 
-import java.util.List;
-
-/**
- * Exception triggered by API requests failed on the server side.
- */
 public class ApiException extends RuntimeException {
 
-  private static final String GLUE_ERROR_MESSAGES = "\n";
-
+  /**
+   */
   private int responseCode;
-  private List<String> messages;
+  private String message;
 
   /**
-   * @param responseCode The HTTP Response code of the failed request.
-   * @param messages The list of messages related to this exception.
+   * @param message The error message of the failed request.
+   * @param responseCode The response code of the failed request.
```

Comment:
These comments can go. So can the whole doc block. We don't need empty doc blocks in java :P